### PR TITLE
Add idr0064 and idr0083

### PR DESCRIPTION
**Pull request by dominikl**
As complete as I could get it... just merge and add missing values in followup PR.
/cc @manics

### CODE REVIEW

File: _data/releases.tsv [hunk 1]
```diff
@@ -34,3 +34,4 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2019-12-10	prod72	0.7.2	11534	1188147	1161986	8370659	61394055	145.97	21.79	398
 2020-01-16	prod73	0.7.3	12544	1188147	1162996	8862811	62870471	159	22.28	423
 2020-03-03	prod80	0.8.0	12569	1195544	1163021	9083954	63754850	169.17	22.68	431
+2020-04-27	prod80			1195994	1163473	9084406	63755752	169.25	22.69
```
Comment (sbesson) [inline _data/releases.tsv:37]: Interestingly, this line captures the fact that we have decoupled IDR releases and IDR deployments i.e. new studies were released twice using the same deployment (`prod80`). The name of the `Data release` the column should probably be modified accordingly e.g.  as `Deployment` or similar.

The empty `Code version` probably raises the more general question of whether we still need a name for referring to a release beyond the `Release date`. Internally we have been using `prod81` for the `2020-04-27` release but we used `prod80`.  Assuming the next IDR release requires a new deployment of the resource, should it be called `prod81`, `prod82` or use a different terminology to dissociate resource releases from individual deployments (`prod80`, `test80`...)?

Comment (manics) [reply _data/releases.tsv:37]: It should be `prod81`. My mistake, I should've noticed that during review.

Comment (sbesson) [reply _data/releases.tsv:37]: Understood so this means we keep the current paradigm or incrementing the `prod` version to refer to new IDR releases. I suspect there is no easy way to update the `version: prodxx{a,b}}` at the bottom of https://idr.openmicroscopy.org/ to reflect this?

Comment (manics) [reply _data/releases.tsv:37]: You could override
- https://github.com/IDR/deployment/blob/91cfc5d079d4a87c74a5f52c7ad1f01fbdfa0b57/ansible/idr-proxy-about.yml#L13
- https://github.com/IDR/deployment/blob/91cfc5d079d4a87c74a5f52c7ad1f01fbdfa0b57/ansible/group_vars/proxy-hosts.yml#L395

**Pull request merged**